src/App.tsx

```diff
@@ -1,24 +1,175 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { ErrNotification } from './Component/ErrNotification/ErrNotification';
+import { Footer } from './Component/Footer/Footer';
+import { Header } from './Component/Header/Header';
+import { TodoList } from './Component/TodoList/TodoList';
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { ErrorType } from './types/ErrorType';
+import { FilterBy } from './types/FilterBy';
 
-const USER_ID = 0;
+const USER_ID = 11091;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  const [activeTodosCount, setActiveTodosCount] = useState<number>(0);
+  const [complitedTodosCount, setComplitedTodosCount] = useState<number>(0);
+
+  const [filterValue, setFilterValue] = useState<FilterBy>(FilterBy.ALL);
+
+  const [errorMessage, setErrorMessage] = useState<ErrorType | null>(null);
+  const [isHidden, setIsHidden] = useState<boolean>(false);
+  const [isProcessings, setIsProcessings] = useState<number[]>([]);
+
+  const showNotification = (value: ErrorType) => {
+    setErrorMessage(value);
+    setTimeout(() => {
```

Comment:
It's a good practice to clear all timers on the component unmount to prevent unpredictable side effects 
https://bobbyhadz.com/blog/react-cleartimeout